keitaroinc/ckanext-datagovmk
# Feature/13.3 outdated datasets

Paster command to be run regularly (preferably every day) to send out notifications for an outdated datasets.

How to test:

Find some dataset that has resources and you have updated a couple of days ago, then make sure that "Frequency/Periodicity" is set to "Daily". This would mean that the dataset resources are updated daily but haven't been updated for a while.

Then run the following paster command:
```
paster --plugin=ckanext-datagovmk check_outdated_datasets -c /etc/ckan/default/development.ini
```

An email should be send to the creator and the Maintainer (if populated) of the dataset with notification.

The command is also available as a CRON script - thus available in the API. To run it, first make sure you are CKAN admin and you have your API Key. Then run (with curl):

```
curl -v http://localhost:5000/api/3/action/datagovmk_start_script -H "Authorization: <API-KEY>" -d '{"name": "outdated_datasets"}'

```
